In Java you can capture any exception object that you caught:

```java
import io.sentry.Sentry;

try {
  aMethodThatMightFail();
} catch(Exception e) {
  Sentry.captureException(e);
}
```

```kotlin
import io.sentry.Sentry

try {
  aMethodThatMightFail()
} catch (e: Exception) {
  Sentry.captureException(e)
}
```
